.github/workflows/test.yml

```diff
@@ -110,100 +110,31 @@ jobs:
 
   test-package:
     name: Test PostgreSQL ${{ matrix.pg }} package install on ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
       matrix:
         pg: ['10', '11', '12', '13', '14']
-        os: [ubuntu-18.04, ubuntu-20.04, ubuntu-latest]
-    env:
-      # Set PATH as postgresql-server-dev-all pretends version is 11
-      PATH: /usr/lib/postgresql/${{ matrix.pg }}/bin:/bin:/usr/bin:/usr/local/bin
+        release: [bionic, focal]
```

Comment:
We need to test on Ubuntu 18.04 as that is what is running in AWS. So does this 'release' array in the matrix cover that off?

Reply:
Yes, Bionic is 18.04.